src/Symfony/Component/Translation/Tests/TranslatorTest.php

```diff
@@ -467,14 +468,18 @@ public function getTransFileTests()
         ];
     }
 
-    public function getTransTests()
+    public function getTransTests(): iterable
     {
-        return [
-            ['Symfony est super !', 'Symfony is great!', 'Symfony est super !', [], 'fr', ''],
-            ['Symfony est awesome !', 'Symfony is %what%!', 'Symfony est %what% !', ['%what%' => 'awesome'], 'fr', ''],
-            ['Symfony est super !', new StringClass('Symfony is great!'), 'Symfony est super !', [], 'fr', ''],
-            ['', null, '', [], 'fr', ''],
-        ];
+        yield ['Symfony est super !', 'Symfony is great!', 'Symfony est super !', [], 'fr', ''];
```

Comment:
please revert to the previous style to free us from potential merge conflicts

Reply:
done